guide/src/componentDocs/CheckableAvatar/CheckableAvatarEx.js

```diff
@@ -8,28 +8,28 @@ class CheckableAvatarEx extends Component {
         super(props);
         this.state = {
             isHovered: false,
-            isChecked: false,
+            isChecked: false
         };
     }
 
     onMouseEnter = () => {
-        const isHovered = true
         this.setState({
-            isHovered,
+            isHovered: true
         });
     };
+
     onMouseLeave = () => {
-        const isHovered = false
         this.setState({
-            isHovered,
+            isHovered: false
         });
     };
+
     onCheck = () => {
-        const isChecked = !this.state.isChecked
+        const isChecked = !this.state.isChecked;
         this.setState({
-            isChecked,
+            isChecked
         });
-    }
+    };
```

Comment:
fwiw, this semi-colon shouldn't be included on this line, they only belong separating statements in a container that supports multiple statements (read a block).In a class definiton, the statements are separated by syntax (key = value)

something a formatter would pick up for us